docs/api.md

```diff
@@ -671,12 +671,51 @@ const result = await page.evaluate(() => {
 console.log(result); // prints "56"
 ```
 
+Passing arguments to pageFunction.
+
+```js
+const name = 'Thomas';
+const age  = 29;
+const location = 'Paris';
+
+const information = await page.evaluate((name, age, location) => {
+  return {
+    name,
+    age,
+    location
+  };
+}, name, age, location);
+
+console.log(information); // prints "{ name: 'Thomas', age: 29, location: 'Paris' }"
+```
+
+Passing both ```pageFunction``` and ```...args``` to ```page.evaluate``` as parameters.
+
+```js
+function manipulateDOM(arg){
+  // manipulate DOM logic
+  return `puppeteer was here and received ${arg}`;
+}
+
+const information = await page.evaluate(manipulateDOM, 42);
```

Comment:
This is more a javascript example then a puppeteer one: as in other examples, we get a function as the first argument and arguments as the rest. I'd remove it.